packages/preact/hooks/src/useShopifyProduct.ts

```diff
@@ -0,0 +1,132 @@
+import { useEffect, useState } from "preact/hooks"
```

Comment:
move this hook to packages/preact/hooks/src/useShopifyProduct/useShopifyProduct.ts